OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -183,5 +172,27 @@ public void SendAirstrike(Actor self, WPos target, bool randomize = true, int at
 				}
 			});
 		}
+
+		private void RemoveCamera(Actor camera)
```

Comment:
Methods are `private` by default, so please drop the keyword.

Reply:
Fixed.